release_notes/osd_latest_product_updates.adoc

```diff
@@ -120,8 +120,9 @@ exampleregistry.net/project/subheading/image:tag
 {product-title} provides a registry to manage container images. The most
 noticeable improvements to the registry are in the user interface.
 
+[discrete]
 [[osd-integrated-registry-ui]]
-*Integrated Registry User Interface*
+===== Integrated Registry User Interface
```

Comment:
@spurtell I made these `[discrete]` headings (won't show up in TOC) because thanks to recent CSS changes, this comes out looking like strong+code. An h5 looks like just bold text, which is what the intention was here originally.